README.md

```diff
@@ -144,6 +144,7 @@ job('upstreamJob') {
             permitAll()
             autoCloseFailedPullRequests()
             displayBuildErrorsOnDownstreamBuilds()
+            commentFilePath("relative/path/to/file")
```

Comment:
Actually, isn't this changed now according to this: https://github.com/jenkinsci/ghprb-plugin/pull/637#issuecomment-388125458 

@bjoernhaeuser ?